### ISSUE TITLE
Add keybinding to switch between fuzzy and nonfuzzy

### ISSUE THREAD

**Issue by nhooyr** [junegunn/fzf#2069]
- [x] I have read through the manual page (`man fzf`)
- [x] I have the latest version of fzf
- [x] I have searched through the existing issues

## Info

- OS
    - [ ] Linux
    - [x] Mac OS X
    - [ ] Windows
    - [ ] Etc.
- Shell
    - [ ] bash
    - [x] zsh
    - [ ] fish
    
## Problem / Steps to reproduce

It'd be nice to a keybinding or an action so that I can set a keybinding to switch between fuzzy and non fuzzy mode.

It's unfortunate to have to go the start of the query string and enter `'` and then go back when I want to make the match unfuzzy quickly.

**Comment by RVSNS** [comment]
create a custom mapping in your shell like `Ctrl + j` to execute `$ fzf -e`(this is exact search)

**Comment by rayiik** [comment]
Hey just a note here you in case you didnt know, you can actually add it in front of any word {wont changr for whole line} but you can mix fuzzy and literal search
![example](https://i.imgur.com/SHtePJI)

**Comment by kaddkaka** [comment]
I agree, it would be nice to have a marker like `'` that you can put at the end of a word to make that word non-fuzzy. Something like:

```
> banana' mlkshk
```

I'm not sure if there are any complications to using `'` for this, but it would be nice to have same character I guess.
